apps/condo/domains/miniapp/tasks/createDefaultB2BAppRoles.js

```diff
@@ -0,0 +1,54 @@
+const { find, getSchemaCtx } = require('@open-condo/keystone/schema')
+const { createTask } = require('@open-condo/keystone/tasks')
+
+const { B2BAppRole } = require('@condo/domains/miniapp/utils/serverSchema')
+
+const SENDER = { dv: 1, fingerprint: 'create-default-b2b-app-role-task' }
+
+async function createDefaultB2BAppRoles (appId, organizationId) {
+    if (!appId || !organizationId) {
+        return
+    }
+
+    const { keystone: context } = await getSchemaCtx('B2BAppRole')
+
+    const permissions = await find('B2BAppPermission', {
+        app: { id: appId },
+        deletedAt: null,
+    })
+    const rolePermissions = Object.assign({}, ...permissions.map(permission => ({
+        [permission.key]: true,
+    })))
+
+    const organizationRoles = await find('OrganizationEmployeeRole', {
+        organization: { id: organizationId },
+        // TODO(DOMA-6750): Include deletedAt at filtering when roles become soft-deletable
+        // deletedAt: null,
+        canManageIntegrations: true,
```

Comment:
Why are the other roles not being created?

Reply:
Because role existence let employee visit miniapp, so we want to keep logic, where only `canManageIntegrations` employees can visit miniapp by default